ajax/libs/tootik/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "tootik",
+  "filename": "tootik.min.css",
+  "version": "1.0.2",
+  "homepage": "https://eliortabeka.github.io/tootik/",
```

Comment:
Let's revise to https://eliortabeka.github.io/tootik/
Because the https://eliortabeka.github.io/tootik/ URL always redirect to that.
Thanks